tests/console/yast2_kdump_setup.pm

```diff
@@ -0,0 +1,32 @@
+# SUSE's openQA tests
+#
+# Copyright 2020 SUSE LLC
+# SPDX-License-Identifier: FSFAP
+
+# Summary: Prepare for yast2 kdump test, include initial test and install packages needed.
+# Maintainer: QE YaST <qa-sle-yast@suse.de>
+
+use base "y2_module_consoletest";
+use strict;
+use warnings;
+
+use registration qw(add_suseconnect_product remove_suseconnect_product);
+use testapi qw(select_console);
+use utils;
+use version_utils 'is_sle';
+
+sub run {
+    select_console('root-console');
+
+    # install kdump by adding additional modules
```

Comment:
this comment should be above `zypper in kdump` but if you write better description, you don't need any of those two comments in the code of this test module. I think those modules are installed only for the libyui-rest-api and this code migh be go away in the future with a proper image with libyui-rest-api pre-installed.

Reply:
ahm ok, now I see the issue, this test module is not needed, we are installing libyui-rest-api in `install_packages_simple`, you just need to put `zypper_call('in kdump')` at the begining of the other 3 test modules you created, we can skip the clean up part of removing the repo.